bionic/protocols.py

```diff
@@ -197,6 +198,29 @@ def __repr__(self):
         return f"{self.__class__.__name__}(...)"
 
 
+class JsonProtocol(BaseProtocol):
+    """
+    Decorator indicating that an entity's values can be serialized using the
+    ``json`` library.
+    """
+
+    def get_fixed_file_extension(self):
+        return "json"
+
+    def validate(self, value):
+        # We don't count sequence types and dict as valid values. Otherwise, we
```

Comment:
I feel like we might as well support lists and dicts here too -- as a user, I would expect to be able to serialize a dict as JSON.